roles/mysql/templates/my.cnf.j2

```diff
@@ -0,0 +1,5 @@
+{{ ansible_managed | comment }}
+
+[client]
+user='root'
+password='{{ mysql_root_password }}'
```

Comment:
Ooh, I'm rusty on this . . . but does this need double-quote as well? `'"{{ mysql_root_password }}"'`? Or does it work as is?

Reply:
works as is.